django/db/backends/base/base.py

```diff
@@ -432,26 +436,34 @@ def validate_no_broken_transaction(self):
     # ##### Foreign key constraints checks handling #####
 
     @contextmanager
-    def constraint_checks_disabled(self):
+    def constraint_checks_disabled(self, table_names):
```

Comment:
For backwards compatibility, should `table_names` be optional? (at least for the backported versions?)